### DISCUSSION TITLE
Clicking on the calendar view opens the item creation screen with the clicked time as start date

### DISCUSSION THREAD

**Discussion by pikaju** [directus/directus#16920] [Feature Requests]
While it is very easy to view and edit items with the calendar view, it is tedious to create new calendar entries, having to manually enter the start and end date in the input fields.

It would be awesome if, similarly to how it works in Google Calendar, clicking on a spot in the calendar opens the item creation screen with some more reasonable values pre-filled. For example, the start date is filled from the date and time you clicked, and the (optional) end date could be an hour later, or something along those lines.

**Comment by azrikahar** [comment]
Great idea! I believe before this can be explored, we will need #13939 to be implemented by #15678 first to achieve the "pre-fill" portion. We can then possibly add the necessary router link in the calendar view with the prefilled query parameters.

**Comment by pikaju** [comment]
Hey @azrikahar, the PR you linked has been closed, and I'm somewhat desperate for this feature :( can I somehow help with the implementation or do anything to speed it up?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
